test/routes/costTime.route.test.js

```diff
@@ -85,13 +78,23 @@ lab.experiment('Cost and time for this permit page tests:', () => {
         const doc = await GeneralTestHelper.getDoc(getRequest)
         checkCommonElements(doc)
         Code.expect(doc.getElementById('includes-waste-recovery-plan')).to.not.exist()
+        Code.expect(GeneralTestHelper.getText(doc.getElementById('length-of-time-text'))).to.equal('up to 13 weeks')
       })
 
       lab.test(`when standard rule is a recovery plan`, async () => {
-        fakeStandardRule.code = 'SR2015 No 39'
+        mocks.standardRule.code = 'SR2015 No 39'
         const doc = await GeneralTestHelper.getDoc(getRequest)
         checkCommonElements(doc)
         Code.expect(doc.getElementById('includes-waste-recovery-plan')).to.exist()
+        Code.expect(GeneralTestHelper.getText(doc.getElementById('length-of-time-text'))).to.equal('up to 13 weeks')
+      })
+
+      lab.test(`when standard rule is for an mcp`, async () => {
+        mocks.standardRuleType.categoryName = 'CATEGORY-NAME-MCP'
```

Comment:
This mock is OK for now as the class isn't currently converting the categoryName to lowercase when using getById, but maybe add in an explicit test to ensure that it also matches 'category-name-mcp'?

Reply:
The mock is still capital letters here but will now be converted to lowercase within the class before being tested against the constants of 'mcpd-sg' and 'mcpd-mcp'